modules/persistent-storage-csi-migration-vsphere.adoc

```diff
@@ -25,62 +23,75 @@ Carefully review the following consequences before opting in to migration:
 * Migration is a disruptive process. The Machine Config Operator (MCO) has to update kubelet, which means degrading each node as it rolls out the new MachineConfig to each machine.
 ====
 
-=== Using the web console to opt in to automatic CSI migration
+Alternatively, if you do not want to upgrade vSphere, you can proceed with an {product-title} upgrade by performing an administrator acknowledgment:
 
-==== Prerequisites
+[source, cli]
+----
+oc -n openshift-config patch cm admin-acks --patch '{"data":{"ack-<version>-api-removals-in-4.14":"true"}}' --type=merge <1>
+----
+<1> Where `<version>` is the {product-title} and associated Kubernetes versions that you are upgrading from: either `4.13-kube-1.27` or `4.12-kube-1.26`.
 
-* Access to the {product-title} web console.
+[IMPORTANT]
+====
+If you do *not* upgrade to vSphere 7.0 Update 3L or 8.0 Update 2 and use an administrator acknowledgment to upgrade to {product-title} 4.14, link:https://access.redhat.com/node/7011683[known issues can occur that can cause the migration to fail].
+====
```

Comment:
> can cause the migration to fail

Migration will succeed regardless but we should word that - one or more pods that use in-tree vSphere volumes may fail to come up in certain cases and additional manual intervention will be necessary for bringing up these pods. Please refer to KCS for more details.

Reply:
+1 problems are with pods that are using these in-tree volumes. I wouldn't care too much about details here, everything is in KCS.
Knows issues can occur, please carefully read the knowledge base article before proceeding with the administrator acknowledgement.